operator/internal/controller/auto/workspace_controller.go

```diff
@@ -97,7 +99,11 @@ func (r *WorkspaceReconciler) Reconcile(ctx context.Context, req ctrl.Request) (
 		w.Status.ObservedGeneration = w.Generation
 		ready.ObservedGeneration = w.Generation
 		meta.SetStatusCondition(&w.Status.Conditions, *ready)
-		return r.Status().Update(ctx, w)
+		err := r.Status().Update(ctx, w)
+		if err != nil {
+			l.Error(err, "updating status")
+		}
+		return err
```

Comment:
This has some additional debug logging because I'm seeing issues where multiple reconcile loops are fighting with each other.

Reply:
Fixed in https://github.com/pulumi/pulumi-kubernetes-operator/pull/678